discovery-infra/test_infra/consts/consts.py

```diff
@@ -124,8 +124,8 @@ class Events:
     IGNITION = "reached installation stage Waiting for ignition"
     CONFIGURING = "reached installation stage Configuring"
     DONE = "reached installation stage Done"
-    CANCELED_CLUSTER_INSTALLATION = "Canceled cluster installation"
-    CANCELED_FOR_HOST = "Installation canceled for host"
+    CANCELED_CLUSTER_INSTALLATION = "Cancelled cluster installation"
+    CANCELED_FOR_HOST = "Installation cancelled for host"
```

Comment:
We should update the variables names as well

Reply:
Don't think so, because we want to change it all later on (here and on the service) to canceled (as it is partially on the service also).